lib/package.c

```diff
@@ -71,6 +71,18 @@ void headerMergeLegacySigs(Header h, Header sigh)
 	case RPMSIGTAG_FILESIGNATURELENGTH:
 	    td.tag = RPMTAG_FILESIGNATURELENGTH;
 	    break;
+	case RPMSIGTAG_VERITYSIGNATURES:
+	    td.tag = RPMTAG_VERITYSIGNATURES;
+	    break;
+	case RPMSIGTAG_VERITYSIGNATURELENGTH:
+	    td.tag = RPMTAG_VERITYSIGNATURELENGTH;
+	    break;
+	case RPMSIGTAG_VERITYSIGNATUREALGO:
+	    td.tag = RPMTAG_VERITYSIGNATUREALGO;
+	    break;
+	case RPMSIGTAG_VERITYSIGNATUREBLKSZ:
+	    td.tag = RPMTAG_VERITYSIGNATUREBLKSZ;
+	    break;
```

Comment:
RPMSIGTAG_FILESIGNATURELENGTH is the wrong example to follow here, for all new tags the signature and main header tags should be the same value so no translation is needed, ie how SHA1/SHA256/RSA/DSA are handled.

Reply:
Fixed, thanks